tools/annovar_av_input.cwl

```diff
@@ -0,0 +1,111 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: kfdrc-annovar-av
+label: Annovar AV Input
+doc: |
+  ANNOVAR annotate av input converted from vcf input
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: ${return inputs.ram * 1000}
+    coresMin: $(inputs.cores)
+  - class: DockerRequirement
+    dockerPull: 'kfdrc/annovar:latest'
+baseCommand: [set,-eo]
+arguments:
+  - position: 1
+    shellQuote: false
+    valueFrom: >-
+      pipefail
+
+      tar xvf $(inputs.cache.path) &&
+      ${
+        var cmd = "";
+        if (inputs.dbscsnv_db) { 
+          cmd += "tar xvf " + inputs.dbscsnv_db.path + " -C humandb && ";
+        }
+        if (inputs.cosmic_db) {
+          cmd += "tar xvf " + inputs.cosmic_db.path + " -C humandb && ";
+        }
+        if (inputs.kg_db) {
+          cmd += "tar xvf " + inputs.kg_db.path + " -C humandb && ";
+        }
+        if (inputs.esp_db) {
+          cmd += "tar xvf " + inputs.esp_db.path + " -C humandb && ";
+        }
+        if (inputs.gnomad_db) {
+          cmd += "tar xvf " + inputs.gnomad_db.path + " -C humandb && ";
+        }
+        if (inputs.run_dbs) {
+          return cmd;
+        } else {
+          return "";
+        }
+      }
+
+      zcat $(inputs.input_av.path) > $(inputs.input_av.nameroot)
```

Comment:
Can table_annovar.pl not handle a zipped input? If not, is it worth it to bgzip it in the first place?

Reply:
It cannot. The outputs are pretty large.  I should revisit this though, because at some point I was worried about jamming these onto to the same machine and running out of space. I guess the only real benefit is if you use the tool as-is, and not part of the pipe, it will save on storage. So, still might be worth the overhead.